src/components/Table/TableData.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from '@xstyled/styled-components'
+import PropTypes from 'prop-types'
+
+import { Typography } from '../'
+
+const TableData = ({ children }) => (
+  <Container>
+    <DataText>{children}</DataText>
+  </Container>
+)
+
+const Container = styled.td`
+  padding: 12px 16px;
+  height: 48px;
+`
+
+const DataText = styled(Typography)`
+  display: flex;
+  flex-direction: row;
+  font-weight: normal;
+  font-size: 16px;
```

Comment:
Aqui entraria o caso que comentei acima
```suggestion
  font-size: 1;
```

Reply:
Removi algumas propriedades aqui já